ton/wallet/wallet.go

```diff
@@ -465,6 +465,32 @@ func (w *Wallet) SendWaitTransaction(ctx context.Context, message *Message) (*tl
 	return w.SendManyWaitTransaction(ctx, []*Message{message})
 }
 
+// BroadcastTransactionsAndWait broadcasts the transaction and waits for confirmation, returning the transaction, block, and any errors.
+func (w *Wallet) BroadcastTransactionsAndWait(ctx context.Context, addr *address.Address, amount tlb.Coins, comment string) (*tlb.Transaction, *ton.BlockIDExt, error) {
```

Comment:
Since this function doing only basic transfer (without custom payload), will be better to call it `TransferWaitTransaction` instead